packages/terra-consumer-layout/src/Layout.jsx

```diff
@@ -12,20 +12,22 @@ const cx = classNames.bind(styles);
 
 const propTypes = {
   /**
-  * Object of configuration for the side navigation and profile.
-  */
-  nav: PropTypes.object.isRequired,
-  /**
-  * Array of links to show for the content of the help button
-  */
-  helpItems: PropTypes.array,
+   * Object of configuration for the side navigation and profile.
+   */
+  nav: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
so @bjankord our rule of thumb is that you only validate the props that you use in the component. So in this case, since nav is just passed to another component, and none of its attributes are accessed, we don't actually validate them. 

I'm curious what you all think and would do cc @mjhenkes @emilyrohrbough @StephenEsser . This is something we are starting to debate internally on our team, but it seems like I might be wrong if the open source linter we use doesn't allow this

Reply:
One option for objects which is used in multiple places is to create a shared prop-types file that all prop-types can reference. I'm not a big fan of this as I think it suffers from out of sight, out of mind, but I've seen people do it in other projects. I'm not really concerned with the way its being done now, I think it is fine.

Reply:
For this instance, I feel disabling the linter could be appropriate if these props are `internal` and passed by the HOC. However, if this is the case, the documentation needs to be updated to indicate this.

IF a consumer is to ever use this prop, this should be split out into a shape object as @bjankord mentions. Given, the reason for this rule is to clean up & clarify documentation for consumers specifying the shape would be good IMO.